src/materials/tdyeos.c

```diff
@@ -6,7 +6,7 @@ static PetscErrorCode ComputeWaterDensity_Constant(PetscReal p, PetscReal *den,
 
   PetscFunctionBegin;
 
-  *den = 998.0;
+  *den = 997.205;
```

Comment:
This will cause a lot of differences in testing. My guess is this change was made by @leorosie to match results for the new salinity model with PFLOTRAN, but we should revert this change.

Reply:
Yes that's correct, you can revert it!